js-exercises/drop-elements/dropElements.js

```diff
@@ -0,0 +1,5 @@
+function dropElements(elements, predicate) {
+  return elements.filter(value => predicate(value));
```

Comment:
@bhupen13au can't we just pass the predicate function?

Reply:
Oh yes we can just pass predicate function directly without calling it